packages/medusa/src/api/routes/admin/users/create-user.ts

```diff
@@ -17,6 +17,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminCreateUserRequest"
+ * x-codegen:
+ *   method: create
```

Comment:
The current JS Client creates an alias to properly type the `role` property from TypeORM. 
The OAS abstracts the problem away by declaring `role?: enum: [admin, member, developer]`.
The codegen should then be able to output the property as `role?: 'admin' | 'member' | 'developer'`.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L64-L67